test/Symbols.test.js

```diff
@@ -0,0 +1,40 @@
+const assert = require('assert');
+const math = require('mathjs');
+
+const print = require('../lib/util/print');
+const Symbols = require('../lib/Symbols');
+
+function runTest(functionToTest, exprString, expectedOutput) {
+  it(exprString + ' -> ' + expectedOutput, function () {
+    const expression = math.parse(exprString);
+    const foundSymbol = functionToTest(expression, 'x');
+    assert.deepEqual(
+      print(foundSymbol),
+      expectedOutput
+    );
+  });
+}
+
+describe('getLastSymbolTerm', function() {
+  const tests = [
+    ['1/x', '1 / x'],
+    ['1/(3x)', '1 / (3x)'],
+    ['3x', '3x'],
+    ['x + 3x', '3x'],
+    ['x/(x+3)', 'x / (x + 3)'],
+  ];
+
+  tests.forEach(t => runTest(Symbols.getLastSymbolTerm, t[0], t[1]));
```

Comment:
can you pass 'x' in too, and then also have some tests where there are multiple symbols and you are looking for one of them? e.g. `'1/x + y', y`